projects/listening-comp/services/transcript-processor/app.py

```
@@ -9,6 +10,13 @@
 from langdetect import detect
 import pandas as pd
 from pytube import YouTube
+from comps import MicroService, ServiceOrchestrator, ServiceType, ServiceRoleType
```

Ruff (0.8.2)

`comps.ServiceRoleType` imported but unused

Remove unused import: `comps.ServiceRoleType`

(F401)